lib/auth.js

```diff
@@ -44,6 +47,17 @@ const getRegistry = opts => {
   return scopeReg || opts.registry
 }
 
+const maybeReadFile = file => {
+  try {
+    return fs.readFileSync(file, 'utf8')
+  } catch (er) {
```

Comment:
It would appear this block isn't covered in tests.

Reply:
Done